frb_example/pure_dart_multi/rust/src/api_block_3.rs

```diff
@@ -0,0 +1,191 @@
+use flutter_rust_bridge::SyncReturn;
```

Comment:
nit: as a general question, shall we test by 2 or 3 blocks? If testing with 3 blocks can catch some types of problems, then definitely we should go with it; but if not, maybe let the test have only 2 blocks will make the future maintainence (e.g. git diff, keep multi file in sync) be a little bit easier